lib/auth/keystore/manager.go

```diff
@@ -93,6 +113,13 @@ func NewManager(cfg Config) (*Manager, error) {
 		backend, err := newPKCS11KeyStore(&cfg.PKCS11)
 		return &Manager{backend: backend}, trace.Wrap(err)
 	}
+	if (cfg.GCPKMS != GCPKMSConfig{}) {
+		if !modules.GetModules().Features().HSM {
+			return nil, trace.AccessDenied("GCP KMS support is only available with an enterprise license")
```

Comment:
Consider using a wrapped version of `auth.ErrRequiresEnterprise` if you can do so without introducing an import cycle.

Reply:
sure, moved the license check out to the auth package